tests/lib/assertions/developer1-pc-18-w-config.model

```diff
@@ -0,0 +1,24 @@
+type: model
```

Comment:
Is this new model used in this PR? It seems like it would be used in  the ubuntu-core-gadget-config-defaults but AFAICT its not. Or am I missing something?